README.rst

```diff
@@ -58,76 +58,110 @@ If you'd like to do a PR we have development instructions `here <https://github.
 List of warnings
 ----------------
 
+.. _B001:
+
 **B001**: Do not use bare ``except:``, it also catches unexpected events
```

Comment:
We may also want to add a link to name so that users can obtain it without having to dig through the source code.